README.md

```diff
@@ -206,6 +206,44 @@ Creating and cloning a build:
 }
 ```
 
+Export options:
+
+Change the export tarball name and cherry pick folders you want to export
+```json
+{
+  "builders": [
+    {
+      "type": "lxc",
+      "target_runlevel": 2,
+      "container_name": "base",
+      "config_file": "lxc.config",
+      "init_timeout": "120s",
+      "template_parameters": [
+        "-d",
+        "ubuntu",
+        "-r",
+        "trusty",
+        "-a",
+        "amd64"
+      ],
+      "template_name": "ubuntu",
+      "export_name": "mytar.tar.gz",
+      "export_folders": [
+        { "src": "/tmp", "dest": "/mytar/temp" }
+      ],
+      "export_permissions": {
+        "owner": "2000",
+        "group": "2000"
+      }
+    }
+  ]
+}
+```
+
+*export_permissions* can only we used when setting export_folders
```

Comment:
Instead of this note, maybe you can make collapse the export related config together:

```
"export": {
  "filename": "some.tar.gz",
  "owner": "2000",
  "group": "2000",
  "folders": [],
}
```
Or whatever naming/grouping makes the most sense to you. Grouping it like this enforces that constraint in the config which I think is better for the user.